<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账单系统</string>
  </property>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>651</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Iosevka</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="money_m">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>月支出：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="money_d">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>日支出：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyin">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>￥</string>
   </property>
  </widget>
  <widget class="QComboBox" name="findmonth">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="findsort">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>餐饮</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>娱乐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>游戏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>日用</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sortin">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>餐饮</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>娱乐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>游戏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>日用</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="eventin">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="money_a">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12334元</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathshow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="pathchoice">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>24</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
